src/select/typeahead/list/typeaheadList.tsx

```diff
@@ -0,0 +1,43 @@
+import "./_typeahead-list.scss";
+
+import React, {Fragment} from "react";
+import classNames from "classnames";
+
+import ListItem from "../../../list/item/ListItem";
+import Tag, {TagShape} from "../../../tag/Tag";
+
+export interface TypeaheadListProps {
+  tags: TagShape[];
+  handleTagRemove: (tag: TagShape) => void;
+  customClassName?: string;
+  input?: React.ReactNode;
+}
+
+function TypeaheadList({
+  tags,
+  customClassName,
+  handleTagRemove,
+  input
+}: TypeaheadListProps) {
```

Comment:
I'm wondering; Why do we need a separated `TypeaheadList` component? Will we be able to use this component as a single?

Reply:
We needed to put the TypeaheadInput into the tag list to improve UX but we had to separate it from the List component. Probably we're not gonna use it outside of the Typeahead.

Reply:
I get it, thank you!